src/test/java/org/folio/rest/api/IsbnNormalizationTest.java

```diff
@@ -0,0 +1,173 @@
+package org.folio.rest.api;
+
+import org.folio.rest.support.builders.RequestItemSummary;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import org.folio.rest.support.Response;
+import static org.folio.util.StringUtil.urlEncode;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.not;
+import static org.folio.rest.api.StorageTestSuite.TENANT_ID;
+import org.folio.rest.support.builders.RequestRequestBuilder;
+import java.util.ArrayList;
+import java.util.UUID;
+import java.util.concurrent.TimeoutException;
+import java.util.concurrent.Callable;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.folio.rest.support.ApiTests;
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+import io.vertx.ext.web.client.HttpResponse;
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Handler;
+import io.vertx.core.buffer.Buffer;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import org.awaitility.Awaitility;
+
+public class IsbnNormalizationTest extends ApiTests {
+  private static final String REQUEST_STORAGE_URL = "/request-storage/requests";
+     // Interesting Times has two ISBNs: 0552167541, 978-0-552-16754-3
+
+  @Before
+  public void setUp() throws InterruptedException, ExecutionException, TimeoutException, MalformedURLException {
+	  createRequests();
+	  Awaitility.await()
+      .atMost(10, TimeUnit.SECONDS)
+      .until(requestsLoaded(), is(true));
```

Comment:
Is the intent of this that the wait until all of the requests have been created?

If so, I don't think this is necessary as the entity creation process only finishes once the request has finished.

Reply:
I assumed so as well at first, but that's not what was actually happening.  Tests would fail, indicating there was no record matching the search parameters.  When I dumped a search for all ISBNS to screen, what I would see was that there were only one or two requests loaded.  So I added the wait and check code-and lo and behold, I started seeing all the records in the body of the test.

Reply:
That is rather strange. I've been trying out some experiments at refactoring these tests (I've only really just started) and I've created a [branch](https://github.com/folio-org/mod-circulation-storage/tree/CIRCSTORE-186-refactor-tests). 

Could you give that code a try and let me know if it runs stably for you.

Reply:
It seems to.  Should I merge it in?

Reply:
> Should I merge it in?

You are welcome to merge it in